scripts/verify_license.py

```diff
@@ -0,0 +1,28 @@
+"""
+ Copyright (C) 2023 Intel Corporation
+
+ Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+ See LICENSE.TXT
+ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+"""
+
+import sys
+import argparse
+
+def verify_file_has_license(file):
+    with open(file, 'r') as in_file:
+        input = in_file.read(300)
```

Comment:
nit: the `input` variable shadows the Python [`input()`](https://docs.python.org/3/library/functions.html#input) builtin, `contents` might be a better name.

Reply:
👍